index.d.ts

```diff
@@ -3,7 +3,7 @@ import { FastifyPlugin, FastifyReply, RawServerBase } from 'fastify';
 declare module "fastify" {
   interface FastifyReply {
     view(page: string, data?: object): FastifyReply;
-    locals?: object;
+    locals?: Record<string, any>;
```

Comment:
```suggestion
    locals?: Record<string, unknown>;
```
Best practice is to avoid `any` as much as possible. Could you please add tests for it?

Reply:
I think any is more appropriate than unknown. Unknown is more for an established API where say a return value could be say null, string, or object and you want to enforce type narrowing. Locals is not an established API and is completely up to the user so any seems more appropriate since the user can't change the type to make it easier to use.

Not sure what you mean by tests. It appears that there are no typescript tests.

Reply:
There is : https://github.com/fastify/point-of-view/blob/master/test/index.test-d.ts

Regarding the `any` type i have no strong feeling on it. Maybe @fox1t have an opinion?